docs/taskruns.md

```diff
@@ -48,8 +48,10 @@ following fields:
   - [`inputs`] - Specifies [input parameters](#input-parameters) and
     [input resources](#providing-resources)
   - [`outputs`] - Specifies [output resources](#providing-resources)
```

Comment:
I wonder why those have `[` and `]` and not `timeout` :thinking:

Reply:
Right. I will add them.